{"url":"https://api.github.com/repos/owncloud/core/issues/775","labels_url":"https://api.github.com/repos/owncloud/core/issues/775/labels{/name}","comments_url":"https://api.github.com/repos/owncloud/core/issues/775/comments","events_url":"https://api.github.com/repos/owncloud/core/issues/775/events","html_url":"https://github.com/owncloud/core/issues/775","id":9105908,"number":775,"title":"oc-2121  -  external storage: dropbox makes owncloud stop working for user under certain circumstances. ","user":{"login":"msrex","id":2625328,"avatar_url":"https://gravatar.com/avatar/24738c0908dad69b83af9cc534bcca80?d=https%3a%2f%2fidenticons.github.com%2ff02814cb1648a9552774c0f932619e91.png&r=x","gravatar_id":"24738c0908dad69b83af9cc534bcca80","url":"https://api.github.com/users/msrex","html_url":"https://github.com/msrex","followers_url":"https://api.github.com/users/msrex/followers","following_url":"https://api.github.com/users/msrex/following{/other_user}","gists_url":"https://api.github.com/users/msrex/gists{/gist_id}","starred_url":"https://api.github.com/users/msrex/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/msrex/subscriptions","organizations_url":"https://api.github.com/users/msrex/orgs","repos_url":"https://api.github.com/users/msrex/repos","events_url":"https://api.github.com/users/msrex/events{/privacy}","received_events_url":"https://api.github.com/users/msrex/received_events","type":"user","site_admin":false},"labels":[{"url":"https://api.github.com/repos/owncloud/core/labels/bug","name":"bug","color":"fc2929"}],"state":"closed","assignee":{"login":"mtgap","id":1097106,"avatar_url":"https://gravatar.com/avatar/454a73ee5feeb428d0c163a48903332c?d=https%3a%2f%2fidenticons.github.com%2f52d29a66f285a1d61125dd69ac0766b2.png&r=x","gravatar_id":"454a73ee5feeb428d0c163a48903332c","url":"https://api.github.com/users/mtgap","html_url":"https://github.com/mtgap","followers_url":"https://api.github.com/users/mtgap/followers","following_url":"https://api.github.com/users/mtgap/following{/other_user}","gists_url":"https://api.github.com/users/mtgap/gists{/gist_id}","starred_url":"https://api.github.com/users/mtgap/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/mtgap/subscriptions","organizations_url":"https://api.github.com/users/mtgap/orgs","repos_url":"https://api.github.com/users/mtgap/repos","events_url":"https://api.github.com/users/mtgap/events{/privacy}","received_events_url":"https://api.github.com/users/mtgap/received_events","type":"user","site_admin":false},"milestone":null,"comments":25,"created_at":"2012-12-08t00:36:47z","updated_at":"2013-03-26t15:25:20z","closed_at":"2013-03-03t00:59:44z","pull_request":{"html_url":null,"diff_url":null,"patch_url":null},"body":"from http://bugs.owncloud.org/thebuggenie/owncloud/issues/oc-2121\nwith a still active reporter.\n\ndescription:\nrecently i added many photos and videos (a total of over 2.5gb) to my dropbox account.\n\nsince then owncloud stopped working for the user that had connected this dropbox account. every time this user tries to access his owncloud account, he gets a server error:\n\n    internal server error\n    the server encountered an internal error or misconfiguration and was unable to complete your request.\n     \n    please contact the server administrator, [no address given] and inform them of the time the error occurred, and anything you might have done that may have caused the error.\n     \n    more information about this error may be available in the server error log.\n\n\n\ni set up a new owncloud installation at a much more powerful server, tried again and the same problem occurs.\n\nother users can use owncloud without problems, only the user with the \"problematic\" dropbox account is affected.\n\nthe only messages i see in owncloud log are like this:\n\n    error files_external error: (400) path must not be empty october 22, 2012, 21:36\n    error files_external error: (400) path must not be empty october 22, 2012, 21:35\n    error files_external error: (400) path must not be empty october 22, 2012, 21:33\n\n\n\nthe solution for the affected user, unless he is the administrator, is for the admin to disable temporary the external storage support, let the user log in, enable the external storage support and the user then proceed to remove his dropbox account before browsing his files. if he is the only administrator, then the problem is fatal.\n\n\nreproduction steps:\nnot sure. best bet is adding a dropbox account with gigabytes of photos and videos.\n","closed_by":{"login":"mtgap","id":1097106,"avatar_url":"https://gravatar.com/avatar/454a73ee5feeb428d0c163a48903332c?d=https%3a%2f%2fidenticons.github.com%2f52d29a66f285a1d61125dd69ac0766b2.png&r=x","gravatar_id":"454a73ee5feeb428d0c163a48903332c","url":"https://api.github.com/users/mtgap","html_url":"https://github.com/mtgap","followers_url":"https://api.github.com/users/mtgap/followers","following_url":"https://api.github.com/users/mtgap/following{/other_user}","gists_url":"https://api.github.com/users/mtgap/gists{/gist_id}","starred_url":"https://api.github.com/users/mtgap/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/mtgap/subscriptions","organizations_url":"https://api.github.com/users/mtgap/orgs","repos_url":"https://api.github.com/users/mtgap/repos","events_url":"https://api.github.com/users/mtgap/events{/privacy}","received_events_url":"https://api.github.com/users/mtgap/received_events","type":"user","site_admin":false}}